libs/core/langchain_core/callbacks/manager.py

```diff
@@ -1954,20 +1953,25 @@ def _configure(
         callback_manager.add_metadata(inheritable_metadata or {})
         callback_manager.add_metadata(local_metadata or {}, False)
 
-    tracer = tracing_callback_var.get()
-    tracing_enabled_ = (
```

Comment:
can we leave dummy vars in they're imported anywhere
```python
tracer = None
tracing_enabled = None
```

Reply:
These are both inside the `_configure` function so I don't think necessary to stub them?